frontend/www/js/omegaup/problem/details.ts

```diff
@@ -75,6 +90,57 @@ OmegaUp.on('ready', () => {
           hasBeenNominated: this.hasBeenNominated,
         },
         on: {
+          'apply-filter': (filter: string, value: string) => {
```

Comment:
```suggestion
          'apply-filter': (filter: 'verdict'|'language'|'username'|'status', value: string) => {
```

así L94-L109 se puede expresar como

```typescript
runsStore.commit('applyFilter', {
 [filter]: value,
} as RunFilters);
```

cortesía de https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015

Reply:
Ohh, que bien. Algo como esto es lo que buscaba, pero no lograba que me funcionara.

Gracias.